pallets/market/src/benchmarking.rs

```diff
@@ -343,13 +343,16 @@ mod benchmarks {
         // Run to 11 to enter pre-commit period
         run_to_block::<T>(11);
 
+        let mut proofs = BoundedVec::new();
+        // Empty proof is considered valid under the DummyProofVerifier
+        proofs.try_push(BoundedVec::new()).unwrap();
+
         // Similar pattern to before
         (0..n)
             // Create ProveCommitSectors from the `n` "index"
             .map(|n| ProveCommitSector {
                 sector_number: n.try_into().unwrap(),
-                // Empty proof is considered valid under the DummyProofVerifier
-                proof: BoundedVec::new(),
+                proofs: proofs.clone(),
```

Comment:
I'm not a fan of this new style because proofs is mut and is getting cloned inside the map, looks kinda weird.

I feel that we could just use the mut inside the map closure, makes it more self contained and the intent clearer

Reply:
I wanted to just use `bounded_vec![bounded_vec![]]` however they're not available in the runtime benchmarks :(